bgpd/bgp_aspath.c

```diff
@@ -1072,7 +1081,7 @@ struct aspath *aspath_aggregate(struct aspath *as1, struct aspath *as2)
 			seg = assegment_append_asns(seg, seg1->as, match);
 
 			if (!aspath) {
-				aspath = aspath_new();
+				aspath = aspath_new(as1->asnotation);
```

Comment:
can we be certain we should conform to `as1` here? should we either check both formats, or allow the caller to specify which format to use?

Reply:
actually, if I follow which function is using aspath_aggregate(), the first parameter is either an aggregate entry of a given bgp instance, or the aspath of an incoming bgp update of a given bgp instance. in both cases, the format has the appropriate sanitation of the given bgp instance.

Reply:
thanks